lib/internal/encoding.js

```diff
@@ -318,8 +317,6 @@ function getEncodingFromLabel(label) {
   return encodings.get(trimAsciiWhitespace(label.toLowerCase()));
 }
 
-const encodeIntoResults = new Uint32Array(2);
```

Comment:
Question: What is the difference between AliasedUint32Array and Uint32Array in v8?

Reply:
The long answer is in aliased_buffer.h. Short answer: it's a wrapper around global typed arrays whose storage is owned by the C++ side with convenient methods for read/write. (A long time ago ChakraCore used it to communicate updates that don't go through the binding for time-travel-debugging but that's not a thing anymore).

Reply:
Thanks!